docs/source/eolearn.features.bands_extraction.rst

```diff
@@ -0,0 +1,7 @@
+eolearn.features.bands_extraction
```

Comment:
Very good that you already created a docs file. But besides this one you also have to update `eolearn.features.rst` file.